lib/compress.js

```diff
@@ -1799,16 +1804,6 @@ merge(Compressor.prototype, {
         }
     };
 
-    OPT(AST_While, function(self, compressor) {
-        if (!compressor.option("loops")) return self;
-        self = AST_DWLoop.prototype.optimize.call(self, compressor);
-        if (self instanceof AST_While) {
-            if_break_in_loop(self, compressor);
```

Comment:
`AST_For` [does `if_break_in_loop()` already](https://github.com/mishoo/UglifyJS2/blob/0610c020b1544820be9898a285ab6c9066490552/lib/compress.js#L1836), so once `AST_While` is turned into `AST_for` and `.transform(compressor)`, it should be covered.

It is technically not calling `if_break_in_loop()` in the same order (or even the same number of times) as before, but I had a scan through the method and thinks this is safe to do if not a minor performance improvement.